src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -1115,11 +1115,14 @@ public KojiBuildInfo getBuildInfo( int buildId, KojiSessionInfo session )
     private KojiBuildInfo withGAV( KojiBuildInfo buildInfo, KojiSessionInfo session )
             throws KojiClientException
     {
+        if (true) return buildInfo;
```

Comment:
I temporarily disabled withGAV on my local to get a clear log to view the data binding...but forgot to remove it when submitting the PR. Sorry guys. Will correct it in no time.